palace/utils/communication.hpp

```diff
@@ -357,7 +358,7 @@ class Mpi
   template <typename... T>
   static void Warning(MPI_Comm comm, fmt::format_string<T...> fmt, T &&...args)
   {
-    Print(comm, "\nWarning!\n");
+    Print(comm, "\n{}\n", fmt::styled("--> Warning!", fmt::fg(fmt::color::yellow)));
```

Comment:
Nice! I did not appreciate that colouring was something we could do with `fmt`.

Reply:
Yeah! I wondered about adding it in more places, for example adding some wrappers for `MFEM_ABORT`/`MFEM_VERIFY`/`MFEM_ASSERT` which print in red. But I think saving that for later is a better idea.